.github/dependabot.yml

```diff
@@ -7,6 +7,17 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: "com.android.tools.build:gradle"
+        update-types: ["version-update:semver-minor"]
```

Comment:
Do we know if ignoring `semver-minor` automatically ignores `semver-patch` as well, or do we have to list both? I would hope it would, but I didn't see anything in the docs that would indicate either way.

Reply:
The docs are unclear, but I now think that maybe they wouldn't, so I'll go ahead and add it.